src/gui/static/src/app/components/pages/settings/outputs/outputs.component.html

```diff
@@ -21,9 +27,6 @@
             <div class="-width-150 -text-right">{{ output.coins | number:'1.0-6' }}</div>
             <div class="-width-150 -text-right -grey">{{ output.calculated_hours | number:'1.0-6' }}</div>
           </div>
-          <div class="-row -small" *ngIf="address.outputs.length === 0">
-            <div class="-flex-fill -hash">{{ 'errors.no-outputs' | translate }}</div>
-          </div>
```

Comment:
This part should not have been removed. Now if you open this page to see the outputs of an address with 0 coins (from the wallets page), the page only shows the address, without the message that indicates that there are no outputs.